irony-cdb-json.el

```diff
@@ -48,8 +48,31 @@ source directory.")
     (get-compile-options (irony-cdb-json--get-compile-options))))
 
 ;;;###autoload
-(defun irony-cdb-json-add-compile-commands-path (project-root
-                                                 compile-commands-path)
+(defun irony-cdb-json--find-common-root-directory (compile-commands-path)
```

Comment:
This part was moved to #239 I believe, I'm still undecided about it, better to leave it there.